app/components/GettingStartedModal/formValidation.ts

```diff
@@ -0,0 +1,20 @@
+import i18n from 'i18next';
+
+import type { GettingStartedSchemaSecret, GettingStartedSecretsData, GettingStartedTechnologiesData } from './model';
+
+export const validateTechnologyFields = (data: GettingStartedTechnologiesData) => {
+    if (!_.some(data, item => item === true)) {
```

Comment:
Maybe simplify to:
```suggestion
    if (!_.some(data, item => item)) {
```
of maybe even
```suggestion
    if (!_.some(data, item)) {
```
I'm not sure the second one is correct, I didn't check it.

Reply:
changed to `if (!_.some(data)) { ...`